language/fixtures/break.rb

```diff
@@ -128,6 +128,19 @@ def break_in_block_in_while
         note :c
       end
     end
+
+    def break_in_another_thread
+      note :a
+      thread = Thread.new do
+        note :b
+        break :break
+        note :c
+      end
+      thread.join rescue nil
+      note :d
+      thread.value
+      note :e
```

Comment:
You're right, thanks for the tip! Fixed in https://github.com/ruby/spec/pull/458/commits/b3b80d66d4a972e64e09c2d3e0792743c2d7a3b5